core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -62,7 +63,8 @@ class LogManager(logDirs: Array[File],
                  val brokerState: BrokerState,
                  brokerTopicStats: BrokerTopicStats,
                  logDirFailureChannel: LogDirFailureChannel,
-                 time: Time) extends Logging with KafkaMetricsGroup {
+                 time: Time,
+                 interBrokerProtocolVersion: ApiVersion) extends Logging with KafkaMetricsGroup {
```

Comment:
This is no longer used.

Reply:
Ah.. I was leaving in a hurry previously. I have updated the patch to revert all unnecessary changes.